styles.css

```diff
@@ -0,0 +1,3 @@
+h1{
+    color:darkred; font-size: 50px; 
+}
```

Comment:
### Although I am highlighting this file, the following changes should be made to the README.md file in this project

- [ ] Kindly use the recommended Microverse README template for this project and all projects that you would do in Microverse. When adopting the template, kindly do the following:

- Remove the Microverse logo. If you have your logo, you can use it, otherwise, I would advise you to remove the logo. 👍 
- Personalize the project name and project summary to the name of your project and a summary of what you implemented. 👍 
- In the **_Tech stack section_** of the README, kindly remove the technologies listed in the `Client`, `Server` and `Database` subsections. Those sections contain technologies that were not used in this project. Kindly remove the `Server` and `Database` subsections and replace those in the `Client` subsections with HTML and CSS. 👍 😎 
- In the **_Key Features section_**, kindly add 1-3 things that you accomplished in this project. I would also advise you to get used to adding these features because you would do that for every project in Microverse and possibly in the professional world. 👍
- If you are not required to deploy the application, you can remove the **_Live Demo section_** of the README.md file. If you are required to deploy, then the link to the deployed project should be pasted under this section. 👍
- In the **_Getting Started Section_**, kindly do the following for the subsections:
  -  In the **_Prerequisite subsection_**, kindly add packages or applications that would be needed to start this application on the local machine. You can include NPM, a browser, a code editor, git and any other thing you can think of. 👍  
  - In the _Setup subsection_, kindly include ways to clone this project and change the directory to the directory of the project. See the example below as a guide:

```sh
  cd my-folder
  git clone git@github.com:myaccount/my-project.git
  cd my-project
```
  - In the **_Install subsection_**, you can indicate ways to install the linters locally and also to run them locally. 👍 😎
  - The _**Usage**_ subsection should contain ways to run the project in the browser if while the Run tests subsections should contain the commands for running the tests written in the project. 👍 😎 
- The **_Authors section_** should contain the social media handles of the owner of the repository and all other collaborators
- The **_Future Features_** section of the README.md file. The Future Features section should contain 1-3 features that can be implemented in the future. Most times, this section acts as a reference to you and future collaborators whenever you want to improve your projects in the future 💻 😎 
- The **_Show Your Support_** Section should communicate to people ways they can help advance the project or give a star to the project if they like it 👍 😎 
- The _**Acknowledgement**_ section should contain information, links and appreciation for those who spurred or contributed to the success of the project 💻 
- The _**FAQ**_ section is the only optional section and it should contain frequently asked questions about the project and answers to those questions 😎 👍 
- The _**LICENSE**_ section should contain permissions given as regards the use of the project. I would advise you to follow the guide on the right side of the article that I would paste below to set a valid LICENSE 👍 

https://choosealicense.com/licenses/mit/